src/components/layouts/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import { LocalIcon, IconName } from "@/assets/icons/local-icon";
+import { LocalImage } from "@/assets/images/local-image";
+import { CompanyProfileCard } from "@/components/ui/companyProfile/companyThumnail/company-profile-card";
+
+
+
+
+const Header = () => {
+  return (
+    <>
+      <div className="flex justify-between items-center py-0 px-[124px] w-full max-w-[2000px]">
+        <div className="flex items-center gap-12 self-stretch">
+          <div className="flex items-center gap-2 self-stretch">
+            <LocalIcon
+              iconName="Logo"
+              width={40}
+              height={40}
+            />
+            <h1 className="font-bold text-2xl">JobHunterly</h1>
+          </div>
+          <div className="gap-4 self-stretch justify-center items-center hidden md:flex">
+            <h1 className="py-2 font-medium text-lg text-[#515B6F]">
+              Find Jobs
+            </h1>
+            <h1 className="border-b-[3px] border-[#4640DE] text-[#4640DE] py-2 font-medium text-lg">
```

Comment:
should use a tailwind.config.js like this:
```tailwind.config.js
            theme: {
  extend: {
    colors: {
      primary: '#4640DE',
    }
  }
}
```
```suggestion
            <h1 className="border-b-[3px] border-primary text-primary py-2 font-medium text-lg"
>
```

Reply:
all codes like this should be fix too